.github/workflows/releaser-signoz.yaml

```
@@ -0,0 +1,66 @@
+      - name: primus-checkout
+        uses: actions/checkout@v4
+        with:
+          repository: signoz/primus
+          ref: ${{ inputs.PRIMUS_REF }}
+          path: .primus
+          token: ${{ steps.token.outputs.token }}
+      - name: info
+        run: |
+          $MAKE info
+      - name: releaser-signoz
+        run: |
```

actionlint (1.7.4)

property "version" is not defined in object type {primus_ref: string; signoz_root_dir: string; signoz_version: string}

(expression)